paimon-web-ui/package.json

```diff
@@ -18,6 +18,9 @@
     "@antv/x6": "^2.15.3",
     "@antv/x6-plugin-dnd": "^2.1.1",
     "@antv/x6-vue-shape": "^2.1.1",
+    "@vicons/fluent": "^0.12.0",
+    "@vicons/material": "^0.12.0",
+    "@vicons/tabler": "^0.12.0",
```

Comment:
Why not use the existing icons to keep the icons consistent?

Reply:
The icons that have been introduced in the project do not have the desired icon, so a new icon library is introduced.